curated_transformers/models/feedforward.py

```diff
@@ -1,30 +1,51 @@
-from typing import Optional
+from typing import Callable, Dict, Optional
 
 import torch
 from torch import Tensor
 from torch.nn import Linear, Module
 
 from .activations import GeluFast, GeluNew
 
+_ACTIVATIONS: Dict[str, Callable[[], Module]] = {
```

Comment:
Can't we use `Type` instead of `Callable[...]`?

Reply:
Sounds good, fixed.